### PR TITLE
Fix bug when 0s are in coverage file

### PR DESCRIPTION
**Pull request by job13er**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [ ] #none# - documentation fixes and/or test additions
 - [x] #patch# - backwards-compatible bug fix
 - [ ] #minor# - adding functionality in a backwards-compatible manner
 - [ ] #major# - incompatible API change

# CHANGELOG
* **Fixed** edge case bug where coverage info includes a `0`.

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -353,14 +353,14 @@ const utils = {
    */
   getCurrentCoverage (config) {
     const coverageSummary = this.readJsonFile(config.features.coverage.file)
-    const totalStatements = __.get(coverageSummary, 'total.statements.total') || -1
+    const totalStatements = __.get(coverageSummary, 'total.statements.total', -1)
```
Comment (job13er) [inline lib/utils.js]: Stop treating `0` the same as `undefined` :)

**Comment by notmessenger** [comment]
👍
